modules/tinymce/src/core/test/ts/browser/DragDropOverridesTest.ts

```diff
@@ -117,4 +117,59 @@ describe('browser.tinymce.core.DragDropOverridesTest', () => {
     ]);
     await pAssertNotification('Dropped file type is not supported');
   });
+
+});
+describe('browser.tinymce.core.DragDropOverridesTest.TINY-8874', () => {
```

Comment:
This seems to be two different sets of tests that doesn't share any code, so I'd say you should either merge the two sets together due to both being for the dragdropoverride or create a separate file for this set. Putting them like this, being entirely separate, just seems like it'd end up being confusing.

Reply:
Even though the tests don't share code, but the tests I made, test stuff that's in the `DragDropOverrides.ts`. So it made sense to me to put the tests in the `DragDropOverridesTest.ts`. Also the reason why I made a new describe block is because I wanted to control the `height` of the editor and it seems I can only set this property when initializing the editor but not after. So I had to create a new describe block to make a new editor hook with the `height` property already set up. What do you think ?

Reply:
Just chiming in quickly here and I haven't reviewed this... anyways Hannes is correct, this isn't what we normally do. Normally we'd use `context` functions inside the `describe` to break it up if possible, otherwise we use a separate test file.

Also as a side note, your "height" won't be working as `innerHeight` is not a valid TinyMCE option.

Reply:
Yeah I figured about the `innerHeight` and I changed it to `height` I just still haven't pushed yet. So if I understand correctly I should use `context` instead of the `describe` I made, and put that inside the existing `describe` block ?

Reply:
Yes, but only if you can move the existing content into a describe since you need a different editor instance. Is there a reason the existing tests can't use a fixed height as well though? I can't see anything about them where that would be an issue.

Reply:
I used height and with and put everything in the same describe block and it worked fine. Thanks 👍

Reply:
Done!